packages/chart/src/components/LineChart/index.tsx

```diff
@@ -207,6 +208,7 @@ const LineChart = (props: LineChartProps) => {
                   <LinePath
                     curve={allCurves[seriesData[0].lineType]}
                     data={
+                      config.visualizationType == "Bump Chart" ? data :
                       config.xAxis.type === 'date-time' || config.xAxis.type === 'date'
                         ? data.sort((d1, d2) => {
                             let x1 = getXAxisData(d1)
```

Comment:
It looks like the data assignment logic is a bit complex and could benefit from some refactoring for better readability and maintainability. Can you try updating to something like this:


```suggestion
const sortedData = (config.xAxis.type === 'date-time' || config.xAxis.type === 'date')
  ? data.sort((d1, d2) => getXAxisData(d1) - getXAxisData(d2))
  : data;

data = config.visualizationType === "Bump Chart" ? data : sortedData;
```